packages/devtools_app/lib/src/screens/memory/shared/primitives/memory_timeline.dart

```diff
@@ -8,12 +8,34 @@ import 'package:intl/intl.dart';
 
 import '../../../../shared/primitives/utils.dart';
 
+class _Json {
+  static const data = 'data';
+}
+
 /// All Raw data received from the VM or offline data.
 class MemoryTimeline {
+  MemoryTimeline({List<HeapSample>? data}) {
```

Comment:
this can be `MemoryTimeline({this.data = const []})`

Reply:
I cannot use const here because this array will grow.
So, i cannot use initializer, so i cannot remove late below. Added comment.